x-pack/plugins/apm/public/components/app/correlations/use_correlations.ts

```diff
@@ -39,7 +34,7 @@ interface RawResponse {
   ccsWarning: boolean;
 }
 
-export const useCorrelations = (params: CorrelationsOptions) => {
+export const useCorrelations = ({ params }: CorrelationsOptions) => {
```

Comment:
What's the purpose of nesting the props inside `params`

Reply:
I think in a past iteration I thought there were more props and I separated those into `params` which would be passed on to the API directly, I removed it again since it's no longer the case in 94aa4f9d98a.